indexer_util/indexer_util/indexer_util.py

```diff
@@ -159,7 +159,7 @@ def bulk_index_docs(es, index_name, docs_by_id):
     # Use generator so we can index arbitrarily large iterators (like tables),
     # without having to load into memory.
     def es_actions(docs_by_id):
-        for _id, doc in docs_by_id:
+        for _id, doc in docs_by_id.iteritems():
             yield ({
                 '_op_type': 'update',
```

Comment:
Let's change _op_type to create. 1) Conceptually we are creating, not updating. 2) This will fail if document already exists